modules/videoio/src/cap_ffmpeg_impl.hpp

```diff
@@ -1153,6 +1155,9 @@ bool CvCapture_FFMPEG::open(const char* _filename, const VideoCaptureParameters&
         CV_LOG_WARNING(NULL, "Unable to read codec parameters from stream (" << _opencv_ffmpeg_get_error_string(err) << ")");
         goto exit_func;
     }
+
+    if(rawMode)
+        return true;
```

Comment:
It breaks some properties like CAP_PROP_FRAME_WIDTH, CAP_PROP_FRAME_HEIGHT, information about camera rotation. Also `video_st` pointer is not initialized.

Reply:
Yes sorry I thought the change would be simple but I was using the the wrong test case.  I'll push the new changes a bit later afte some more testing.